docs/human-interface-guidelines.md

```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
 
-App launcher icons have three primary goals:
+There are three aspects to note when designing an elementary icon:
 
-1. Promote the brand and tell the story of the app.
-2. Help users discover the app in the software center.
-3. Function well in the OS.
+* Its baseline (highlighted in <span style="color:blue;">blue</span>) to ensure that all icons of one size line up along the bottom when in a row (much like text).
+* Its mean line height (<span style="color:green;">green</span>), also known as the center line of your canvas.
+* The x-height (shown in <span style="color:red;">red</span>) or "how tall" your icon is.
 
-#### Promote the brand story {#promote-the-brand-story}
+<img class="hig-icons-example" style="background-image:url(/images/docs/human-interface-guidelines/icons/example-icon1.png);" src="/images/docs/human-interface-guidelines/icons/grid.svg">
+<img class="hig-icons-example" style="background-image:url(/images/docs/human-interface-guidelines/icons/example-icon2.png);" src="/images/docs/human-interface-guidelines/icons/grid.svg">
 
-App launcher icons are an opportunity to showcase the brand and hint at the story of what your app is about. Thus, you should:
+Keeping these lines in mind while designing, means you can place elements along them so icons appear more consistent when put together. For example, notice how some elements in both the Terminal and Videos icon above relate to mean line.
 
-* Create an icon that is unique and memorable.
-* Use a color scheme that suits your brand.
-* Don't try to communicate too much with the icon. A simple icon will have more impact and be more memorable.
-* Avoid including the application name in the icon.
+### Common Measurements
 
-#### Help users discover the app in the software center {#help-users-discover-the-app-in-the-software-center}
+| Canvas Size   | Base Line     | x-Height | Mean Line Height |
+| ------------- |:------------- | -------- | ---------------- |
+| 16x16         | 1             | 14       | 8                |
+| 24x24         | 2             | 20       | 12               |
+| 32x32         | 2             | 26       | 16               |
+| 48x48         | 3             | 40       | 24               |
+| 64x64         | 4             | 56       | 32               |
+| 128x128       | 9             | 104      | 64               |
 
-App launcher icons are the first look that prospective users will get of your app in the software center. A high quality app icon can influence users to find out more as they scroll through lists of apps.
+### Exceptions
 
-Quality matters here; a well-designed icon can be a strong signal that your app is of similarly high quality. Consider working with an icon designer to develop the app’s launcher icon.
+However there are exceptions. Many icons (especially mimetype icons) have ascending and descending elements -those that extend beyond the base line and x-height line (shown here in <span style="color:orange;">orange</span>.)
```

Comment:
Probably should just be a colon since it's not interjecting right? Otherwise I guess it would be an em dash